README.md

```diff
@@ -1,44 +1,49 @@
-[![CI](https://img.shields.io/github/actions/workflow/status/libp2p/js-http-v1-content-routing/js-test-and-release.yml?branch=main\&style=flat-square)](https://github.com/libp2p/js-http-v1-content-routing/actions/workflows/js-test-and-release.yml?query=branch%3Amain)
+[![codecov](https://img.shields.io/codecov/c/github/libp2p/js-delegated-routing-v1-http-api-content-routing.svg?style=flat-square)](https://codecov.io/gh/libp2p/js-delegated-routing-v1-http-api-content-routing)
+[![CI](https://img.shields.io/github/actions/workflow/status/libp2p/js-delegated-routing-v1-http-api-content-routing/js-test-and-release.yml?branch=main\&style=flat-square)](https://github.com/libp2p/js-delegated-routing-v1-http-api-content-routing/actions/workflows/js-test-and-release.yml?query=branch%3Amain)
 
-> Use a Routing V1 HTTP service to discover content providers
+> Use a Delegated Routing V1 HTTP service to discover content providers
+
+This is a [ContentRouting](https://libp2p.github.io/js-libp2p/interfaces/_libp2p_interface.content_routing.ContentRouting.html)
+implementation that makes use of the [@helia/delegated-routing-v1-http-api-client](https://www.npmjs.com/package/@helia/delegated-routing-v1-http-api-client)
+to use servers that implement the snappily-titled [Delegated Routing V1 HTTP API](Delegated Routing V1 HTTP API)
+spec to get/put IPNS records and to resolve providers for CIDs.
 
 ## Table of contents <!-- omit in toc -->
 
-- [Install](#install)
-  - [Browser `<script>` tag](#browser-script-tag)
+- - [Install](#install)
+    - [Browser `<script>` tag](#browser-script-tag)
 - [Example](#example)
-- [API Docs](#api-docs)
-- [License](#license)
-- [Contribution](#contribution)
+  - [API Docs](#api-docs)
+  - [License](#license)
+  - [Contribution](#contribution)
 
 ## Install
 
 ```console
-$ npm i @libp2p/http-v1-content-routing
+$ npm i @libp2p/delegated-routing-v1-http-api-content-routing
 ```
 
 ### Browser `<script>` tag
 
-Loading this module through a script tag will make it's exports available as `Libp2pHttpV1ContentRouting` in the global namespace.
+Loading this module through a script tag will make it's exports available as `Libp2pDelegatedRoutingV1HttpApiContentRouting` in the global namespace.
 
 ```html
-<script src="https://unpkg.com/@libp2p/http-v1-content-routing/dist/index.min.js"></script>
+<script src="https://unpkg.com/@libp2p/delegated-routing-v1-http-api-content-routing/dist/index.min.js"></script>
 ```
 
-## Example
+# Example
 
 ```js
 import { createLibp2p } from 'libp2p'
-import { reframeContentRouting } from '@libp2p/reframe-content-routing'
+import { delgatedRoutingV1HTTPAPIContentRouting } from '@libp2p/delegated-routing-http-v1-content-routing'
 
 const node = await createLibp2p({
   contentRouters: [
-    reframeContentRouting('https://cid.contact/reframe')
+    delgatedRoutingV1HTTPAPIContentRouting('https://example.org')
```

Comment:
When https://github.com/protocol/bifrost-community/issues/1 lands (EOW week I expect), delegated-ipfs.dev will be a good default.